tests/custom/flatten_lex/tree_core/find_v_find/matrix_2_bools/stdout.expected

```diff
@@ -0,0 +1,79 @@
+language ESSENCE' 1.0
+
+find A_1: matrix indexed by [int(0..1)] of bool
+find A_2: matrix indexed by [int(0..1)] of bool
+find B_1: matrix indexed by [int(0..1)] of bool
+find B_2: matrix indexed by [int(0..1)] of bool
+branching on [A_1, A_2, B_1, B_2]
+such that
+    flatten([flatten([[-toInt(A_1[q3]); int(1)], [-toInt(A_2[q3]); int(1)]; int(1..2)]) | q3 : int(0..1)]) <lex
+    flatten([flatten([[-toInt(B_1[q4]); int(1)], [-toInt(B_2[q4]); int(1)]; int(1..2)]) | q4 : int(0..1)])
+
+$ Conjure's
```

Comment:
Same here with the comment lines.

Reply:
I think I will have to go through the lot and apply a filter. Not a problem though.

Reply:
If you want a generic filter see: https://github.com/conjure-cp/conjure/blob/master/tests/exhaustive/acceptOutput.sh#L19-L20